src/Writer/Xml/NilEmptyWriter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace DemandwareXml\Writer\Xml;
+
+class NilEmptyWriter
+{
+    private $writer;
+
+    public function __construct(XmlWriter $writer)
+    {
+        $this->writer = $writer;
+    }
+
+    public function writeElement($name, $content = null): bool
```

Comment:
Is name deliberately not type hinted?

Reply:
Yeah - I copied `NilEmptyWriter` from `NotEmptyWriter`

Reply:
Ah yeah, I dragged it down and see they're just aping the methods they're calling